style/less/layout/preferences/prefs_shortcuts.less

```
@@ -120,4 +128,9 @@
       display: none;
     }
   }
+  .shc_val {
+		&::placeholder {
+			color: transparent;
+		}
+  }  
```

Merge into single selector when only one rule.
```suggestion
  .shc_val::placeholder {
    color: transparent;
  }  
```